starlingcaptureapi/exif.py

```diff
@@ -56,6 +56,46 @@ def convert_timestamp(self, value_str):
         """
         return parser.parse(value_str).strftime("%Y:%m:%d %H:%M:%S %z")
 
+    def extract_from_exif_data(self, exif_data):
+        """Returns the latitude and longitude, if available, from the provided exif_data (obtained through get_exif_data above)"""
+        lat = None
+        lon = None
+
+        if "GPSInfo" in exif_data:
+            gps_info = exif_data["GPSInfo"]
+
+            gps_latitude = gps_info.get("GPSLatitude")
+            gps_latitude_ref = gps_info.get(gps_info, 'GPSLatitudeRef')
+            gps_longitude = gps_info.get(gps_info, 'GPSLongitude')
+            gps_longitude_ref = gps_info.get(gps_info, 'GPSLongitudeRef')
+
+            if gps_latitude and gps_latitude_ref and gps_longitude and gps_longitude_ref:
+                lat = self._to_degress_from_rational_str(gps_latitude)
+                if gps_latitude_ref != "N":
+                    lat = 0 - lat
+
+                lon = self._to_degress_from_rational_str(gps_longitude)
+                if gps_longitude_ref != "E":
+                    lon = 0 - lon
+
+        return lat, lon
+
+    def _to_degress_from_rational_str(self, value):
```

Comment:
Value is an array of tuples, so a better name for this method could be `_to_deg_from_rational_tuples`, or something like that.